src/api/data/scheduledMessage/scheduledMessage.mutations.ts

```diff
@@ -36,5 +41,23 @@ export const mutationResolvers = {
       pubsub.publish(subscriptions.SCHEDULEDMESSAGE_ADDED, scheduledMessage)
       return scheduledMessage
     },
+    async reviewRequest(obj, { messageData }, context, info) {
+       const data: IMailerHTML<any> = {
+        to: 'tpll5wiiajplpxd6@ethereal.email',
```

Comment:
This should come from function args. Add a field for the to email in `ReviewRequestData`.

Reply:
Ok! I'll include on the params and in the interface